bin/setup

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+# Exit if any subcommand fails
+set -e
```

Comment:
This is copied from another of our projects, and this is how we do it in all of our other `bin/setup` scripts are done